src/components/shared/form/index.jsx

```diff
@@ -0,0 +1,44 @@
+import { StyledDiv } from 'components/shared/layout';
+import PropTypes from 'prop-types';
+import { tuna } from 'theme/colors';
+import CheckboxComponent from './CheckboxComponent';
+import LabelComponent from './LabelComponent';
+import LegendComponent from './LegendComponent';
+import FieldsetComponent from './FieldsetComponent';
+/**
+ * @name LabeledCheckbox
+ * @description A component used to display styled checkbox element.
+ *
+ * @param {bool}	[disabled]		: Whether the checkbox is disabled.
+ * @param {bool}	[checked]		: Whether the checkbox is checked.
+ * @param {string}	id				: The id of the checkbox.
+ * @param {string}	children		: text to display in the label.
+ */
+const LabeledCheckbox = ({ children, id, checked, disabled, ...otherProps }) => (
+	<StyledDiv {...otherProps} display="flex" alignItems="center">
+		<CheckboxComponent disabled={disabled} checked={checked} id={id} />
```

Comment:
How do we give it an `onChange` event ?

Reply:
`{...otherProps}` moved to `CheckboxComponent`